### ISSUE TITLE
Tripod Mech Center Leg Actuators cannot be Salvaged (Nor damaged via GM Tools)

### ISSUE THREAD

**Issue by dlegalov** [MegaMek/mekhq#1236]
### Environment
Version: 0.46.0-rc.1
OS: Windows 7 64-bit
Java: 1.8.0_191

### Description
A custom (file attached) tripod mech was added to the roster, and the center leg was
damaged through GM Tools. When attempting to repair the center leg, the component is a generic "Mech Location":
![image](https://user-images.githubusercontent.com/11690997/62100873-2afb1c00-b248-11e9-89a4-00fd7adfeeca.png)
![image](https://user-images.githubusercontent.com/11690997/62100947-77def280-b248-11e9-8dc1-afe9f0bad139.png)
After acquiring the component, the leg components must be salvaged:
![image](https://user-images.githubusercontent.com/11690997/62101083-094e6480-b249-11e9-9f40-cc8df740f093.png)
However, while the DHS was salvaged,
none of the center leg actuators show up for salvaging:
![image](https://user-images.githubusercontent.com/11690997/62101138-3a2e9980-b249-11e9-80e0-459d3b88d769.png)
Looking at the GM Tool damage editor, the center leg actuators are not accessible:
![image](https://user-images.githubusercontent.com/11690997/62101177-58949500-b249-11e9-9d86-21441a0242dc.png)
In the warehouse, the center leg is a generic location:
![image](https://user-images.githubusercontent.com/11690997/62101212-7c57db00-b249-11e9-84ca-e6ae254b6ce5.png)

In summary, a destroyed Tripod center leg cannot be fully stripped out, and thus cannot be
repaired without GM Tools.
The center leg is also not marked as such in the parts warehouse.
I cannot mark the leg actuators destroyed, so I cannot test what happens if they are damaged.
The use case for this was offline campaign management, so damage is only applied through GM Tools.

### Files
[megameklog.txt](https://github.com/MegaMek/mekhq/files/3445154/megameklog.txt)
[mekhqlog.txt](https://github.com/MegaMek/mekhq/files/3445155/mekhqlog.txt)
[Test Campaign31500102.zip](https://github.com/MegaMek/mekhq/files/3445157/Test.Campaign31500102.zip)
Custom Mech used in testing
[Emperor Crab 00000.zip](https://github.com/MegaMek/mekhq/files/3445163/Emperor.Crab.00000.zip)
